src/main/request.ts

```diff
@@ -22,7 +22,7 @@ export const DEFAULT_REQUEST_CONFIG: RequestConfig = {
     auth: new NoAuthAgent(),
     retryAttempts: 10,
     retryDelay: 500,
-    statusCodesToRetry: [[401, 401], [429, 429], [500, 599]],
```

Comment:
Hm, might be a good idea to keep retrying on 502 and 503. What do you think?

Reply:
Yup, after reading this, might be a good idea to keep 429 too:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After

Maybe in the future, we can even check the res header and adjust the retry interval :P